test/unit/spell.test.js

```diff
@@ -1,45 +1,49 @@
 'use strict';
 
 const assert = require('assert').strict;
-const path = require('path');
-
-const { connect, Bone } = require('../..');
-const Post = require('../models/post');
-const TagMap = require('../models/tagMap');
-const Comment = require('../models/comment');
-const Book = require('../models/book');
-const User = require('../models/user');
-const Realm = require('../..');
-
-before(async function() {
-  await connect({
-    models: path.resolve(__dirname, '../models'),
-    database: 'leoric',
-    user: 'root',
-    port: process.env.MYSQL_PORT,
-  });
-});
+
+const { connect, raw, Bone } = require('../..');
+
+class Post extends Bone {
+  static table = 'articles'
+  static initialize() {
+    this.hasOne('attachment', { foreignKey: 'articleId' });
+  }
+}
+class TagMap extends Bone {}
+class Comment extends Bone {}
+class Book extends Bone {}
+class User extends Bone {}
+class Attachment extends Bone {}
```

Comment:
stopped requiring fixture models in unit tests because unit tests now runs in the a single mocha session to reduce mocha boot time.

This should also makes running tests in parallel in the future much easier.